src/python/dxpy/dx_extract_utils/somatic_filter_payload.py

```diff
@@ -33,6 +32,33 @@
     "tumor_normal": "is",
 }
 
+extract_utils_basepath = os.path.join(
+    os.path.dirname(dxpy.__file__), "dx_extract_utils"
+)
+
+chrs = []
+for i in range(1, 23):
+    chrs.append(str(i))
+chrs.extend(["Y", "X"])
+
+def retrieve_geno_bins(list_of_genes, project, genome_reference, friendly_name):
+    """
+    A function for determining appropriate geno bins to attach to a given SYMBOL, Gene or Feature
+    """
+    stage_file = "Homo_sapiens_genes_manifest_staging_vep.json"
+    platform_file = "Homo_sapiens_genes_manifest_vep.json"
+    error_message = "Following symbols, genes or features are invalid"
+    genome_reference = genome_reference + "_" + friendly_name
+    geno_bins = retrieve_bins(list_of_genes, project, genome_reference, extract_utils_basepath,
```

Comment:
Do we support variable bin size here?
I tought that this method returns bins equivalent to the bins in the database, instead this returns list of gene positions like:
`
... ,
{
    "chr": "1",
    "start": "1471765",
    "end": "1497848",
    "strand": "+"
  },
...
`
Right? 
So maybe we can reffer to this as get genomics intervals?

Reply:
`geno_bins` is a key in the payload for `RAW` API. Named the variable as `geno_bins` to be consistent with that.